### PR TITLE
fix: colgroup child count

### PR DESCRIPTION
**Pull request by nschonni**

### CODE REVIEW

**Review by sideshowbarker — APPROVED** [review]

File: files/en-us/mdn/editor/keyboard_shortcuts/index.html [hunk 1]
```diff
@@ -11,6 +11,8 @@
 <table class="standard-table">
  <colgroup>
   <col style="width: 15em;">
+  <col>
+  <col>
```
Comment (vkWeb) [inline files/en-us/mdn/editor/keyboard_shortcuts/index.html:15]: Looks like one extra `col` here.

**Review by vkWeb — CHANGES_REQUESTED** [review]

File: files/en-us/mdn/editor/keyboard_shortcuts/index.html (hunk 1 above)
Comment (nschonni) [reply files/en-us/mdn/editor/keyboard_shortcuts/index.html]: Thanks! I must have gone and fixed it twice 😆

Comment (vkWeb) [reply files/en-us/mdn/editor/keyboard_shortcuts/index.html]: @nschonni happens :rofl:

**Review by vkWeb — APPROVED** [review]
Now let's merge! :tada:

**Review by chrisdavidmills — APPROVED** [review]

**Pull request merged**